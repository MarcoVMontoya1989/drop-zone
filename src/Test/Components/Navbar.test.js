import {shallow} from "enzyme";
import {render, screen} from '@testing-library/react';
import NavbarComponent from "../../Components/Component/Navbar.component";

describe('Navbar Tests', function () {

  const wrapper = shallow(<NavbarComponent />);

  it('should render component', function () {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render text Drop Zone', function () {
    render(<NavbarComponent />);

    const DropText = screen.getByText('Drop');
    const ZoneText = screen.getByText('Zone');

    expect(DropText).toBeInTheDocument();
    expect(ZoneText).toBeInTheDocument();
  });
});
