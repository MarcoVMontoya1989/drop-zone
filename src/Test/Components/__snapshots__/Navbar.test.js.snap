// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Tests should render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="navbar"
      >
        <div
          class="container"
        >
          <div
            class="navbar-end navbar-end-force"
          >
            <a
              class="navbar-item navbar-item-drop"
            >
              <div
                class="columns drop-zone"
              >
                <div
                  class="column drop-content"
                  id="rohza-text"
                >
                  Drop
                </div>
                <div
                  class="column drop-content drop-content-end"
                  id="rohza-text"
                >
                  Zone
                </div>
              </div>
            </a>
          </div>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="navbar"
    >
      <div
        class="container"
      >
        <div
          class="navbar-end navbar-end-force"
        >
          <a
            class="navbar-item navbar-item-drop"
          >
            <div
              class="columns drop-zone"
            >
              <div
                class="column drop-content"
                id="rohza-text"
              >
                Drop
              </div>
              <div
                class="column drop-content drop-content-end"
                id="rohza-text"
              >
                Zone
              </div>
            </div>
          </a>
        </div>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
